A31-xds-routeaction-support.md

```diff
@@ -150,41 +150,45 @@ cluster name provided to it, which should not be possible when following the
 steps above in that order, it will drop the RPC with an INTERNAL status code.
 
 The following drawing shows which parts of the system will contain which current
-Load Balancing Policies.  The blue box represents the Config Selector and the
-red box represents the top-level Load Balancing Policy and its children.
+Load Balancing Policies.
 
 ![xDS Architecture](A31_graphics/xds_architecture.png "xDS Architecture")
 
 #### Supported Fields
 
-Initially, only the timeout-related `RouteAction` settings of
-[`max_stream_duration.max_stream_duration`](https://github.com/envoyproxy/envoy/blob/6f2ad057f8655e2688a195337f7520dbd77015fa/api/envoy/config/route/v3/route_components.proto#L775)
-and
-[`max_stream_duration.grpc_max_timeout`](https://github.com/envoyproxy/envoy/blob/6f2ad057f8655e2688a195337f7520dbd77015fa/api/envoy/config/route/v3/route_components.proto#L781)
-will be supported.  gRPC will not support
-[`timeout`](https://github.com/envoyproxy/envoy/blob/6f2ad057f8655e2688a195337f7520dbd77015fa/api/envoy/config/route/v3/route_components.proto#L952)
-and
-[`max_stream_duration.grpc_timeout_offset`](https://github.com/envoyproxy/envoy/blob/6f2ad057f8655e2688a195337f7520dbd77015fa/api/envoy/config/route/v3/route_components.proto#L789).
+Only some xDS timeout-related settings will be supported:
+
+- [`RouteActions.max_stream_duration.max_stream_duration`](https://github.com/envoyproxy/envoy/blob/6f2ad057f8655e2688a195337f7520dbd77015fa/api/envoy/config/route/v3/route_components.proto#L775)
+- [`RouteActions.max_stream_duration.grpc_max_timeout`](https://github.com/envoyproxy/envoy/blob/6f2ad057f8655e2688a195337f7520dbd77015fa/api/envoy/config/route/v3/route_components.proto#L781)
+- [`max_stream_duration`](https://github.com/envoyproxy/envoy/blob/6f2ad057f8655e2688a195337f7520dbd77015fa/api/envoy/config/core/v3/protocol.proto#L95) from the [`HttpConnectionManager.common_http_options`](https://github.com/envoyproxy/envoy/blob/6f2ad057f8655e2688a195337f7520dbd77015fa/api/envoy/extensions/filters/network/http_connection_manager/v3/http_connection_manager.proto#L289) as a default if `RouteActions.max_stream_duration.max_stream_duration` is unset.
+
+Notably, gRPC will not support
```

Comment:
How about making this a bulleted list, just like the one above where you list the fields that we *do* support?  That way, we can discuss each of these fields individually.

Reply:
Done